ci/pipeline

```diff
@@ -168,11 +174,19 @@ def build(): String = {
   }
 
   val version = createTarballPackages()
+  buildDockerAndLinuxPackages()
+  version
+}
+
+def buildDockerAndLinuxPackages(): Unit = {
   utils.stage("Package Docker Image, Debian and RedHat Packages") {
     %('sbt, "docker:publishLocal", "packageLinux")
-  }
 
-  version
+    // create test docker images and tests packages
+    val testPath = pwd/'tests/'package
+    %('make, "all")(testPath)
+    %('amm, "test.sc", "all")(testPath)
```

Comment:
Hm, I don't think `test.sc` would be found. Should it be something like `%('amm, testPath / "test.sc", "all")(testPath)`.

Also, it seems weird to call ammonite from ammonite. I'd actually expect `make all test`. That seems cleaner to me. What do you think?

Reply:
1st... it works as written.   the `(testPath)` sets the working directory... which took some time for me to figure out.   As this is the working dir of the invocation the `test.sc` script all is good.

2nd.   I was codifying what our manual release process has been.    We could add a test target to the Makefile.   I discussed the need to call ammonite from ammonite with @timcharper seemed like the thing to do.    Changing it to make adds a layer of indirection but doesn't change the fact we are calling amm from amm.
I'm good the way it is... but totally willing to change it if you feel strongly about it.

Reply:
Thanks for enlighten me on the work path. That makes sense now.

As to the ammonite call I'm on the fence. To me this file is like a meta build tool that triggers other build tools such as sbt and flake8. In that sense the build tool for testing the Docker image would be `make`. However, I don't think it's a blocker.

Reply:
@jeschkies that's not a bad idea! We could create a "make test" target which builds all necessary docker images and then runs the tests.

Reply:
Looks like that was already done :)